docs/userGuide/components/searchbar.md

```diff
@@ -0,0 +1,23 @@
+# Searchbar
+
```

Comment:
Could provide a brief description mentioning about how it can be used.
```
A searchbar allows visitors of your website to quickly find items matching any of the keywords typed in.

To use a searchbar in a navigation bar, see [Use searchbar]({{baseUrl}}/userguide/contentauthoring#use-searchbar)
```